src/science/landuse/landuse3.F90

```diff
@@ -3,7 +3,23 @@ MODULE landuse_variable
   IMPLICIT NONE
 
   SAVE
-
+!*# Overview of landuse3.F90
+!
+! these are three-type landuse-type variable
+!
+! luc%var_x(mland,mvmax): variable indexed by "mland" and "mvmax" before landuse change
+! luc%var_y(mland,mvmax): variable indexed by "mland" and "mvmax" after land use change
+! lucmp%var(mp):          variable indexed by the patch number from 1 to mp
```

Comment:
```suggestion
! luc%var_x(mland,mvmax): variable indexed by "mland" and "mvmax" before landuse change
!
! luc%var_y(mland,mvmax): variable indexed by "mland" and "mvmax" after land use change
!
! lucmp%var(mp):          variable indexed by the patch number from 1 to mp
```
This will put all of these lines on their own lines in the documentation.